lib/active_admin_sidebar.rb

```diff
@@ -1,5 +1,5 @@
 require 'active_admin'
-require "active_admin_sidebar/version"
+require 'active_admin_sidebar/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.